Library/Homebrew/cask/dsl.rb

```diff
@@ -311,17 +311,18 @@ def version(arg = nil)
     #
     # ```ruby
     # sha256 arm:   "7bdb497080ffafdfd8cc94d8c62b004af1be9599e865e5555e456e2681e150ca",
-    #         intel: "b3c1c2442480a0219b9e05cf91d03385858c20f04b764ec08a3fa83d1b27e7b2"
+    #        intel: "b3c1c2442480a0219b9e05cf91d03385858c20f04b764ec08a3fa83d1b27e7b2"
+    #        linux: "1a2aee7f1ddc999993d4d7d42a150c5e602bc17281678050b8ed79a0500cc90f"
```

Comment:
Does this imply linux x86_64? Given linux arm will be coming in the near future: what do you invisage doing there? `linux_intel` or something may make more sense

Reply:
I considered this, but it feels weird:
```rb
sha256 macos:
         arm: "abc"
       intel: "def"
```

Reply:
feel like having some Linux-specific makes more sense until the majority of casks support Linux

Reply:
shall I take the bottle naming? `x86_64_linux` and `arm64_linux`?

Reply:
@SMillerDev yeh, that would work for me. honestly, feels like it'd have been nicer if this whole DSL did the same instead of `intel` 👍🏻